svc-bie-kafka/src/main/java/gov/va/vro/services/bie/model/BieMessagePayload.java

```diff
@@ -0,0 +1,27 @@
+package gov.va.vro.services.bie.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import lombok.extern.jackson.Jacksonized;
+
+@JsonInclude
+@Setter
+@Getter
+@Builder
+@Jacksonized
+@ToString
+public class BieMessagePayload {
+
+  @JsonProperty("topic")
```

Comment:
I don't think you need all these extraneous `@JsonProperty` since they are the same as the field name
```suggestion
```